mlflow/utils/model_utils.py

```diff
@@ -263,3 +265,39 @@ def _validate_pyfunc_model_config(model_config):
             "Values in the provided ``model_config`` are of an unsupported type. Only "
             "JSON-serializable data types can be provided as values."
         )
+
+
+def _validate_model_assignment_in_init(cls):
```

Comment:
Can we rename this function? `validate` sounds like an exception is thrown if `__init__` doesn't have `self.model = ...`.

Reply:
how about `check`?

Reply:
`check` sounds good.